### ISSUE TITLE
Error when user doesn't change details

### ISSUE THREAD

**Issue by tgorringe4** [nhsbsa/tlrs-prototype#13]
As you know if someone doesn’t have their certificate number they’re asked to confirm their name, dob and address. If they answer yes to all of these questions they aren’t updating anything, so my assumption is that they won’t go to the ‘Check your answers’ screen but will go straight to a decision (after check new details on the diagram), probably ‘not found’.
 
We know that the diagram attached needs updating, but there’s no ‘Yes’ line from ‘Is this how your address appears’, and when I’ve tried to do this journey using the PPC reference you end up with an internal server error screen, so I’m unable to confirm to Venkata where you should go from this page.

We have decided that they should still go to the check your answers page that will then take them on to the alternative options before landing on the can't find page.

**Comment by tgorringe4** [comment]
[full service SR (26.02.18).pdf](https://github.com/nhsbsa/tlrs-prototype/files/1758191/full.service.SR.26.02.18.pdf)

This diagram shows the updated journey

**Comment by tgorringe4** [comment]
How can I replicate this in the prototype? With cert-number you can enter a number that will decide whether the exemption can be found but with this route there is no way of specifying what the end result will be. 

One solution is to just pick one outcome. It shouldn't matter too much as we can still test the end pages in different journeys instead. The other solution, which I can do in parallel is to create the journey with the other end point on the static prototype.

One last solution is to build a variable for the end page into each scenario. This can be selected using the PCN ref no. of the scenario. If this solution were used the check answers page would just have a snippet of Js to determine where to point the user.

If the user hasnt changed any details though it should always end at 'not found.'

**Comment by tgorringe4** [comment]
Changed the answers-handler. It's now much simpler. If no details are changed it sends the user from check your answers to dwp-exemptions. I also changed check-address-handler so that if they have a Mat or Med they will still be directed to the change details page after not changing their address details. This means we can ask every possibility before asking if they were preg or med.

I have a new question that may lead into a new issue ticket. If a user doesn't change details then should we send them to the BSA options before the DWP options incase there is another BSA exemption they could try instead?

Also fix the check answers so that it doesn't show the contact details part of the table when it's just a case of not changing any details.
